spec/lib/publishing_api_finder_publisher_spec.rb

```diff
@@ -15,6 +15,7 @@
             "content_id" => SecureRandom.uuid,
             "format" => "a_report_format",
             "signup_content_id" => SecureRandom.uuid,
+            "logo_path" => "https://assets.digital.cabinet-office.gov.uk/media/5540ab8aed915d15d8000030/esi_funds_eu_logo.gif",
```

Comment:
Updated with a more generic path. It still needs to be present otherwise it fails validation